client/app/cloud/project/compute/cloud-project-compute.html

```diff
@@ -47,6 +47,14 @@
         </div>
 
         <div class="cloud-compute-content-ctnr-responsive">
+            <div class="cloud-compute-message-container">
+                <oui-message data-ng-hide="CloudProjectComputeCtrl.isInfoMessageDismissed()"
+                    data-type="info"
+                    data-dismissed="CloudProjectComputeCtrl.isInfoMessageDismissed()"
+                    data-on-dismiss="CloudProjectComputeCtrl.dismissInfoMessage()">
+                    <span data-ng-bind-html="'cpc_message' | translate"></span>
```

Comment:
cpc_message should be splitted in 2
Le Cloud OVH arrive en Allemagne !
Découvrez notre datacentre de Limburg.

That way we wouldn't be obligated to put the <a> tag in the message.. which is nice.  However I can see you tried to be generic about your implementation so I'm not sure